src/zui/ZUIOrganizeSidebar.tsx

```diff
@@ -233,6 +241,72 @@ const ZUIOrganizeSidebar = (): JSX.Element => {
             })}
           </List>
         </Box>
+        <Box
+          sx={{
+            bottom: 0,
+            position: 'absolute',
+            width: '100%',
+          }}
+        >
+          <Divider />
+          <Box
+            sx={{
+              display: 'flex',
+              justifyContent: 'space-between',
+              mx: open ? 1 : 0.5,
+              my: 0.5,
+              py: open ? 1.25 : 1,
+            }}
+          >
+            {user && (
+              <>
+                <Box
+                  sx={{
+                    alignItems: 'center',
+                    display: 'flex',
+                    justifyContent: open ? 'flex-start' : 'center',
```

Comment:
Correct me if I'm wrong, but this is different than the solution we used for the other parts of the menu, right? There we assigned a fixed width for the icon (avatar in this case) and kept it as always `center`. I may be remembering this incorrectly or misunderstanding the context here so let me know if you disagree.

Reply:
Yes! ZUIAvatar will not accept a width property but instead I´ve added it in the Box that wraps it and remove the justifyContent and it works! I hope I understood you right 🌡️